agents/chess-agent.json

```
@@ -0,0 +1,51 @@
+{
+  "protocolVersion": "0.3.0",
+  "author": "Telex",
+  "wellKnownURI": "https://system-integration.telex.im/chessagent/.well-known/agent.json",
+  "name": "Chess Agent",
+  "description": "An agent that plays chess. Accepts moves in standard notation and returns updated board state as FEN and an image.",
+  "url": "https://system-integration.telex.im/chessagent",
+  "version": "1.0.0",
+  "provider": {
+    "organization": "Telex",
+    "url": "https://www.telex.im"
+  },
+  "documentationUrl": "https://system-integration.telex.im/chessagent/docs",
+  "capabilities": {
+    "streaming": false,
+    "pushNotifications": false,
+    "stateTransitionHistory": true
+  },
+  },
```

```suggestion
  },
```

you have an extra parenthesis